[Docs] update references and metrics post Mysticeti launch (#19198)
MystenLabs/sui: docs/content/guides/operator/validator-committee.mdx

The process of collecting validator signatures on a transaction into a certificate and the process of submitting certificates can be performed in parallel. The client can simultaneously multicast transactions/certificates to an arbitrary number of validators. Alternatively, a client can outsource either or both of these tasks to a third-party service provider. This provider must be trusted for liveness (it can refuse to form a certificate), but not for safety (it cannot change the effects of the transaction, and does not need the user's secret key).

> ### Certificates / certified transactions

Reviewer: ```suggestion
### Certificates / certified transactions {#certificates}
```

If a client collects a quorum of signatures on the effects of the transaction, then the client has a promise of finality. This means that transaction effects persist on the shared database and are actually committed and visible to everyone by the end of the epoch. This does not mean that the latency is a full epoch, because you can use the effects certificate to convince anyone of the transactions finality, as well as to access the effects and issue new transactions. As with transactions, you can parallelize the process of sharing a certificate with validators and (if desired) outsource to a third-party service provider.

## The role of consensus

> Sui uses a high-throughput DAG-based consensus engine implementing the Mysticeti algorithm to sequences certified transactions. Consensus produces a sequence of commits. In each commits, there is a sequence of certified transactions.

Reviewer: ```suggestion
Sui uses a high-throughput DAG-based consensus engine implementing the Mysticeti algorithm to sequences certified transactions. Consensus produces a sequence of commits. In each commit, there is a sequence of certified transactions.
```

> The order of transactions sequenced by consensus determines the relative order they can operate on each shared object. Note that executions of transactions touching different shared objects are parallelized on multiple cores.

Reviewer: ```suggestion
The order of transactions that consensus sequences determines the relative order they can operate on each shared object. Note that executions of transactions touching different shared objects are parallelized on multiple cores.
```
